packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -143,83 +139,78 @@ const AmountValue = ({
       color={amountValueColor}
       lineHeight={amountLineHeights[type][size]}
     >
-      {value}
+      {amount.formatted}
```

Comment:
Earlier `<Amount value={123456.789} suffix="humanize">` was rendering `123.46K`, but now it renders only `123K` ( or `1.2L`). Is this formatting expected? not showing decimals for `K` but showing it in case of `L`.

Reply:
@snitin315 It will render `123.46K` for `en-US` locale but `1.2L` for `en-IN` locale. 
The decimals are omitted based on the locale and rounding off of digits.